homeassistant/components/fronius/quality_scale.yaml

```diff
@@ -0,0 +1,87 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  appropriate-polling: done
+  brands: done
+  common-modules:
+    status: done
+    comment: |
+      Single platform only, so no entity.py file.
+      CoordinatorEntity is used.
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup:
+    status: exempt
+    comment: |
+      This integration does not subscribe to events.
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  config-entry-unloading: done
+  docs-configuration-parameters:
+    status: exempt
+    comment: |
+      This integration does not provide configuration options.
+  docs-installation-parameters: done
+  entity-unavailable: done
+  integration-owner: done
+  log-when-unavailable: done
+  parallel-updates:
+    status: done
+    comment: |
+      Coordinators are used and asyncio.Lock mutex across them ensure proper
+      rate limiting. Platforms are read-only.
+  reauthentication-flow:
+    status: exempt
+    comment: |
+      This integration doesn't require authentication.
+  test-coverage: done
+  # Gold
+  devices: done
+  diagnostics: done
+  discovery-update-info: done
+  discovery: done
+  docs-data-update: done
+  docs-examples: done
+  docs-known-limitations: done
+  docs-supported-devices: done
+  docs-supported-functions: done
+  docs-troubleshooting: done
+  docs-use-cases: done
+  dynamic-devices: done
+  entity-category: done
+  entity-device-class: done
+  entity-disabled-by-default: done
+  entity-translations: done
```

Comment:
You can remove
```json

      "frequency_ac": {
        "name": "[%key:component::sensor::entity_component::frequency::name%]"
      },
```
as it will use device class translations

Reply:
I think this goes the same for
```json

      "power_apparent": {
        "name": "[%key:component::sensor::entity_component::apparent_power::name%]"
      },
      "power_factor": {
        "name": "[%key:component::sensor::entity_component::power_factor::name%]"
      },
```

Reply:
```json

      "power_real_ac": {
        "name": "[%key:component::sensor::entity_component::power::name%]"
      },
      "temperature_channel_1": {
        "name": "[%key:component::sensor::entity_component::temperature::name%]"
      },
```

Reply:
done in #132876